code-challenges.test.js

```diff
@@ -13,7 +13,7 @@
 // --------------------1) Prompt: Create a function that takes in a number (greater than 2) and returns an array containing the Fibonacci sequence. The length of the array is determined by the argument of the function.
 
 // a) Create a test with expect statements for each of the variables provided.
-
+// 
```

Comment:
Challenge 1: The test variables are numbers, which means the argument for your function calls should be a number, not an object.